helpers/ipfs.ts

```diff
@@ -47,3 +47,7 @@ export function buildMetadata(
 
   return data;
 }
+
+export function transformURL(url: string) {
+  return url.replace("ipfs://", "https://ipfs.infura.io/ipfs/");
```

Comment:
Will these move to environment variables at some point?

Reply:
It's not required for now.